book/book.toml

```diff
@@ -6,6 +6,10 @@ src = "src"
 title = "A/B Street"
 
 [output.html]
+mathjax-support = true # equations if necessary
```

Comment:
```suggestion
```
We can add the mathjax and any other plugin when we first use it.
I haven't looked up what `site-url` does, but if we weren't using it before and things were fine, not sure there's value adding this note